packages/components/src/components/icon/common/ic-poinc-failed.svg

```diff
@@ -0,0 +1,18 @@
+<svg width="96" height="116" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Minify this file using svgo command.
Ref: packages/components/stories/icon/README.md